components/IscnRegisterForm.vue

```diff
@@ -850,6 +866,17 @@ export default class IscnRegisterForm extends Vue {
     return 'CreativeWork'
   }
 
+  get defaultLanguage() {
+    const containsChinese = (str: any) => /[\u4E00-\u9FFF\u3400-\u4DBF\uF900-\uFAFF]/.test(str);
```

Comment:
Suggest to just simply check for all ascii/alphanumeric for English and avoid unicode logic